Jenkinsfile_CNP

```diff
@@ -6,7 +6,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
 def product = "civil"
 def component = "ccd"
-def camundaBranch = "master"
+def camundaBranch = "feat/CIV-4306_Payment_of_hearing_fee"
```

Comment:
This will need to be reverted prior to merging but I think that's known.

Marking the review as "Request changes" to prevent merging by mistake with this value not reset.